MailerQ/MessageStore/MessageStorage.cs

```diff
@@ -6,13 +6,16 @@
 
 namespace MailerQ.MessageStore
 {
+    /// <summary>
+    /// MailerQ External Message Storage
+    /// </summary>
     public class MessageStorage : IMessageStorage
     {
         private readonly IMessageStorage storageEngine;
 
-        public MessageStorage(IOptions<MailerQConfiguration> options)
+        public MessageStorage(MailerQConfiguration configuration)
```

Comment:
I am not sure if it is a good idea to accept resolved options along with the alternative with IOptions.

_IOptions_ alternative is more powerful, it could allow resolving the values at the moment that they are required, supporting form example hot updates.